gax/src/test/java/com/google/api/gax/rpc/OperationCallableImplTest.java

```diff
@@ -0,0 +1,983 @@
+/*
```

Comment:
Why is this test marked as new not moved? It is important for this one, because it is a huge and important test, which test stuff, which was heavily modified as part of this PR. It is hard to review it porperly, when everything is green.

Reply:
It's weird, it seems like the files should be close enough to match. Darn git! 

Maybe I'll create a "Diff PR" like I did for a previous large PR.